Packet++/header/HttpLayer.h

```diff
@@ -207,201 +207,217 @@ namespace pcpp
-			Http207MultiStatus,
+			Http207MultiStatus = 207,
 			/** 208 Already Reported */
-			Http208AlreadyReported,
+			Http208AlreadyReported = 208,
+			/** 229-225 Unassigned */
 			/** 226 IM Used */
-			Http226IMUsed,
+			Http226IMUsed = 226,
+			/** 227-299 Unassigned */
+			
 			/** 300 Multiple Choices */
-			Http300MultipleChoices,
+			Http300MultipleChoices = 300,
 			/** 301 Moved Permanently */
-			Http301MovedPermanently,
-			/** 302 (various messages) */
-			Http302,
+			Http301MovedPermanently = 301,
+			/** 302 Found */
+			Http302Found = 302,
 			/** 303 See Other */
-			Http303SeeOther,
+			Http303SeeOther = 303,
 			/** 304 Not Modified */
-			Http304NotModified,
+			Http304NotModified = 304,
 			/** 305 Use Proxy */
-			Http305UseProxy,
-			/** 306 Switch Proxy */
-			Http306SwitchProxy,
+			Http305UseProxy = 305,
+			/** 306 Unused */
 			/** 307 Temporary Redirect */
-			Http307TemporaryRedirect,
+			Http307TemporaryRedirect = 307,
 			/** 308 Permanent Redirect, */
-			Http308PermanentRedirect,
+			Http308PermanentRedirect = 308,
+			/** 309-399 Unassigned */
+
 			/** 400 Bad Request */
-			Http400BadRequest,
+			Http400BadRequest = 400,
 			/** 401 Unauthorized */
-			Http401Unauthorized,
+			Http401Unauthorized = 401,
 			/** 402 Payment Required */
-			Http402PaymentRequired,
+			Http402PaymentRequired = 402,
 			/** 403 Forbidden */
-			Http403Forbidden,
+			Http403Forbidden = 403,
 			/** 404 Not Found */
-			Http404NotFound,
+			Http404NotFound = 404,
 			/** 405 Method Not Allowed */
-			Http405MethodNotAllowed,
+			Http405MethodNotAllowed = 405,
 			/** 406 Not Acceptable */
-			Http406NotAcceptable,
+			Http406NotAcceptable = 406,
 			/** 407 Proxy Authentication Required */
-			Http407ProxyAuthenticationRequired,
+			Http407ProxyAuthenticationRequired = 407,
 			/** 408 Request Timeout */
-			Http408RequestTimeout,
+			Http408RequestTimeout = 408,
 			/** 409 Conflict */
-			Http409Conflict,
+			Http409Conflict = 409,
 			/** 410 Gone */
-			Http410Gone,
+			Http410Gone = 410,
 			/** 411 Length Required */
-			Http411LengthRequired,
+			Http411LengthRequired = 411,
 			/** 412 Precondition Failed */
-			Http412PreconditionFailed,
+			Http412PreconditionFailed = 412,
 			/** 413 RequestEntity Too Large */
-			Http413RequestEntityTooLarge,
+			Http413RequestEntityTooLarge = 413,
 			/** 414 Request-URI Too Long */
-			Http414RequestURITooLong,
+			Http414RequestURITooLong = 414,
 			/** 415 Unsupported Media Type */
-			Http415UnsupportedMediaType,
+			Http415UnsupportedMediaType = 415,
 			/** 416 Requested Range Not Satisfiable */
-			Http416RequestedRangeNotSatisfiable,
+			Http416RequestedRangeNotSatisfiable = 416,
 			/** 417 Expectation Failed */
-			Http417ExpectationFailed,
-			/** 418 I'm a teapot */
-			Http418ImATeapot,
-			/** 419 Authentication Timeout */
-			Http419AuthenticationTimeout,
-			/** 420 (various messages) */
-			Http420,
+			Http417ExpectationFailed = 417,
+			/** 418 Unused **/
```

Comment:
don't add "unused" and "unassigned" code number into the list

Reply:
`418` is `I'm a teapot` message: https://en.wikipedia.org/wiki/Hyper_Text_Coffee_Pot_Control_Protocol

Reply:
answered above, let's discussed in the above thread